python/core/qgsvectorlayer.sip

```diff
@@ -685,6 +685,11 @@ class QgsVectorLayer : QgsMapLayer, QgsExpressionContextGenerator
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
+     */
+    virtual bool deleteStyleFromDatabase( const QString& styleId, QString &msgError );
```

Comment:
You still need to leave the `/Out/`annotation in place or the error message will not be accessible in python